feat: Added support for Romanian locale

stefy opened the pull request:
<!--
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]
-->

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

<!--
1. Describe the source of requirement, like related issue link.
-->

### 💡 Background and solution

Added support for Romanian locale

### 📝 Changelog

<!--
Describe changes from userside, and list all potential break changes or other risks.
--->

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Added support for Romanian locale         |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed


-----
[View rendered docs/react/i18n.en-US.md](https://github.com/stefy/ant-design/blob/feature/docs/react/i18n.en-US.md)
[View rendered docs/react/i18n.zh-CN.md](https://github.com/stefy/ant-design/blob/feature/docs/react/i18n.zh-CN.md)

stefy committed:
Added support for Romanian locale

afc163 commented on docs/react/i18n.zh-CN.md:
```diff
@@ -65,6 +65,7 @@ return (
 | 泰米尔人         | ta_IN  |
 | 泰语             | th_TH  |
 | 土耳其语         | tr_TR  |
+| 罗马尼亚         | ro_RO  |
```
罗马尼亚语

stefy replied:
Fixed

afc163 commented:
Please add test case like other language and update snapshots.

https://ant.design/docs/react/contributing#Sending-a-Pull-Request

stefy commented:
> Please add test case like other language and update snapshots.
> 
> https://ant.design/docs/react/contributing#Sending-a-Pull-Request

Added test cases for Romanian locales and updated the snapshots. Thanks!
